CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- ⚠️ The default database directory moved to `/var/lib/vast` for Linux deployments.
```

Comment:
@dominiklohmann were you thinking to use the *breaking change* category for this?

Reply:
No. It's not breaking for existing installations.